ocs_ci/utility/utils.py

```diff
@@ -1071,6 +1072,11 @@ def get_testrun_name():
             {key: '-' for key in ''' \\/.:*"<>|~!@#$?%^&'*(){}+`,=\t'''}
         )
     )
+    # form complete testrun name that includes deployment platform, type and runid
+    testrun_name = (
+        f"{testrun_name} - {config.ENV_DATA.get('platform').upper()}-"
+        f"{config.ENV_DATA.get('deployment_type').upper()} (RUN ID: {config.RUN['run_id']})"
```

Comment:
*RUN ID* should be left in the Subject as it was. The output from `get_testrun_name()` is used for Polarion reporting and for the same configuration/test suite it should return the same value. RUN ID is changing for each build and if it will be part of the testrun name, it will lead to creating new "test plan" in Polarion for each run (not grouping the same runs together).

Reply:
Done